tests/bootstrap.php

```diff
@@ -2,7 +2,6 @@
 
 error_reporting(E_ALL);
 
-require __DIR__ . '/../vendor/autoload.php';
```

Comment:
we should not 'default' in tests to use psr4, we should use classes from the `_generated.php` file. On the other hand, we have a tests in the `auto-instrumentation` that tests usage of `_generated.php` and composer's `psr4`.